doc/tutorial/Tutorial.tex

```diff
@@ -664,10 +664,10 @@ \subsection{Existential quantification}
 Lemma refl_if : forall x : D, (exists y, R x y) -> R x x.
 \end{coq_example}
 
-Remark that the hypotheses which are local to the currently opened sections
+The hypotheses that are local to the currently opened sections
```

Comment:
https://www.youtube.com/watch?v=N1aWStx9O2Q

(that being said, the manual has chosen to use a standard American style, therefore, it is reasonable to indeed use _that_ rather than _which_ here)